cmd/serf/command/agent/config.go

```diff
@@ -363,7 +367,9 @@ func containsKey(keys []string, key string) bool {
 func MergeConfig(a, b *Config) *Config {
 	var result Config = *a
 
-	// Copy the strings if they're set
+	//TODO(schristoff): do i need to check nodename
```

Comment:
I think we can remove this todo (since we're verifying the final config that gets passed to Create() anyway)